settings.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencyResolutionManagement {
     repositories {
         google()
         mavenCentral()
+        maven { url = uri("https://jitpack.io") }
```

Comment:
I don't see any issues with this. However, could you clarify why we use JitPack.io?

Reply:
I had to add this to use a [RatingBar Library](https://github.com/a914-gowtham/compose-ratingbar) and it needed this addition on the gradle settings to be able to work correctly :)

Reply:
Okay yes it does make sense then !